### PR TITLE
Update About.mdx

### PR DESCRIPTION
**Pull request by samanthahamilton**
Issue #687

## Related issues and PRs

-n/a

## Description

-Improved styling to optimize for reading (like a blog)
-Improve sentence structure and paragraphs

## Impacted Areas in the application

-/about page

## Testing

-n/a

## Additional Notes
-.docx file attached with comments for copyedit changes
-Can continue with other pages after review

[neherlabCOVID19_Review_SH_26Jul2020.docx](https://github.com/neherlab/covid19_scenarios/files/4978303/neherlabCOVID19_Review_SH_26Jul2020.docx)

### CODE REVIEW

**Comment by ivan-aksamentov** [comment]
@samanthahamilton Hi Samantha, thanks. It looks great!
I think there's just a little bug in a comment below. Could you please take a look?

File: src/pages/About.mdx [hunk 1]
```diff
@@ -14,48 +14,40 @@ import LinkExternal from '../components/Router/LinkExternal'
...
- - infectious individuals recover or progress to severe disease. The ratio of recovery to severe progression depends on age
-
- - severely sick individuals either recover or deteriorate and turn critical. Again, this depends on the age
-
- - critically ill individuals either get admitted to ICU (if space is available), or are placed in an overflow compartment. Younger age-groups are given preferential access to ICU.
-
- - critically ill individuals either return to regular hospital or die. Again, this depends on the age and on whether they receive intensive care or not.
-
+  * Susceptible individuals are exposed/infected through contact with infectious individuals. Each infectious individual causes on average $R_0$ secondary infections while they are infectious.
+    - Transmissibility of the virus could have seasonal varia
```
Comment (ivan-aksamentov) [inline src/pages/About.mdx:28]: This sentence seems to be cut short, compared to the old one

Comment (samanthahamilton) [reply src/pages/About.mdx]: You are right, my apologies. I have made changes to fix it; hopefully, it's resolved!
Cheers.

Comment (ivan-aksamentov) [reply src/pages/About.mdx]: @samanthahamilton Thanks!
It looks good to me. I will just let Richard @rneher to review and merge if he's okay with the changes.

File: src/pages/About.mdx [hunk 2]
```diff
@@ -14,48 +14,40 @@ import LinkExternal from '../components/Router/LinkExternal'
...
-
- - severely sick individuals either recover or deteriorate and turn critical. Again, this depends on the age
-
- - critically ill individuals either get admitted to ICU (if space is available), or are placed in an overflow compartment. Younger age-groups are given preferential access to ICU.
-
- - critically ill individuals either return to regular hospital or die. Again, this depends on the age and on whether they receive intensive care or not.
-
+  * Susceptible individuals are exposed/infected through contact with infectious individuals. Each infectious individual causes on average $R_0$ secondary infections while they are infectious.
+    - Transmissibility of the virus could have seasonal variation which is parameterized with the parameter "seasonal forcing" (amplitude) and "peak month" (month of most efficient transmission).
+  * Exposed individuals progress to a symptomatic/infectious state after an average latency. This progression happens in three stages to ensure the distribution of times spent in the exposed compartment is more realistic than a simple exponential model.
```
Comment (rneher) [inline src/pages/About.mdx:29]: "...average latency." -> "...latency period."

File: src/pages/About.mdx [hunk 3]
```diff
@@ -73,10 +65,11 @@ Many respiratory viruses such as influenza, common cold viruses (including other
 ## Transmission reduction
 
 The tool allows one to explore temporal variation in the reduction of transmission by infection control measures.
-This is implemented as a curve through time that can be dragged by the mouse to modify the assumed transmission. The curve is read out and used to change the transmission relative to the base line parameters for $R_0$ and seasonality.
+This is implemented as a curve through time that can be dragged by the mouse to modify the assumed transmission. The curve is read out and used to change the transmission relative to the base line parameters for $R_0$ and seasonality. 
+
```
Comment (rneher) [inline src/pages/About.mdx:69]: Ohhh, this is outdated. 
"Control measures can be specified as time intervals with start and end date during which transmission is reduced by a certain amount."

File: src/pages/About.mdx [hunk 4]
```diff
@@ -109,16 +102,17 @@ $$
 
 </div>
 
-The parameters of this model fall into three categories: a time dependent infection rate $\beta(t)$ time scales of transition to a different subpopulation $t_l$, $t_i$, $t_h$, $t_c$, and age specific parameters $m_a$, $c_a$ and $f_a$ that determine relative rates of different outcomes.
-The latency time from infection to infectiousness is $t_l$, the time an individual is infectious after which he/she either recovers or falls severely ill is $t_i$, the time a sick person recovers or deteriorates into a critical state is $t_h$, and the time a person remains critical before dying or stabilizing is $t_c$.
-The fraction of infectious that are asymptomatic or mild is $m_a$, the fraction of severe cases that turn critical is $c_a$, and the fraction of critical cases that are fatal is $f_a$.
+The parameters of this model fall into three categories: (1) A time dependent infection rate $\beta(t)$ time scales of transition to a different subpopulation $t_l$, $t_i$, $t_h$, $t_c$, and age specific parameters $m_a$, $c_a$ and $f_a$ that determine relative rates of different outcomes;
```
Comment (rneher) [inline src/pages/About.mdx:105]: there needs to be a comma after "$\beta(t)$"

File: src/pages/About.mdx [hunk 5]
```diff
@@ -109,16 +102,17 @@ $$
...
-The latency time from infection to infectiousness is $t_l$, the time an individual is infectious after which he/she either recovers or falls severely ill is $t_i$, the time a sick person recovers or deteriorates into a critical state is $t_h$, and the time a person remains critical before dying or stabilizing is $t_c$.
-The fraction of infectious that are asymptomatic or mild is $m_a$, the fraction of severe cases that turn critical is $c_a$, and the fraction of critical cases that are fatal is $f_a$.
+The parameters of this model fall into three categories: (1) A time dependent infection rate $\beta(t)$ time scales of transition to a different subpopulation $t_l$, $t_i$, $t_h$, $t_c$, and age specific parameters $m_a$, $c_a$ and $f_a$ that determine relative rates of different outcomes; 
+(2) The latency time from infection to infectiousness is $t_l$, the time an individual is infectious after which he/she either recovers or falls severely ill is $t_i$, the time a sick person recovers or deteriorates into a critical state is $t_h$, and the time a person remains critical before dying or stabilizing is $t_c$;
```
Comment (rneher) [inline src/pages/About.mdx:106]: This isn't really a 1,2,3... what is now under (1) is an overview, the two following points give detail. Pretty confusing wording on our part...

File: src/pages/About.mdx (hunk 3 above)
Comment (samanthahamilton) [reply src/pages/About.mdx:69]: Is a better alternative to remove line 76 (68 in the edit) entirely, and replace it with "Control measures can be specified as time intervals with start and end date during which transmission is reduced by a certain amount."?

File: src/pages/About.mdx (hunk 5 above)
Comment (samanthahamilton) [reply src/pages/About.mdx:106]: I believe I understand this section more fully now. How about this edit...?

"The parameters of this model fall into three categories, which determine the relative rates of different outcomes: (1) A time-dependent infection rate  [B(t)]; (2) Time scales of transition to a different subpopulation [tl], [ti], [th], [tc]; and (3) Age-specific parameters [ma]. [ca], and [fa]. Here the latency time from infection to infectiousness is presented as [tl], the time an individual is infectious after which he/she either recovers or falls severely ill is [ti], the time a sick person recovers or deteriorates into a critical state is [th], and the time a person remains critical before dying or stabilizing is t[c]. While the fraction of infectious individuals that are asymptomatic or mild is [ma], the fraction of severe cases that turn critical is [ca], and the fraction of critical cases that are fatal is [fa]." 

Hopefully, this more clearly explains the parameter categories and the variables within them?

File: src/pages/About.mdx (hunk 3 above)
Comment (rneher) [reply src/pages/About.mdx:69]: yes! that is what I meant.

File: src/pages/About.mdx (hunk 5 above)
Comment (rneher) [reply src/pages/About.mdx:106]: yes, I like this!

**Comment by ivan-aksamentov** [comment]
@samanthahamilton Merging this. Please keep up the great work!

**Pull request merged**